### PR TITLE
Post update

### PR DESCRIPTION
**Pull request by zldzksk1**
I updated the bookpost page

### CODE REVIEW

**Review by ravishruthi — APPROVED** [review]
Not a show stopper, but maybe add 'rejected' variable in swap object?

**Comment by zldzksk1** [comment]
> Not a show stopper, but maybe add 'rejected' variable in swap object?

Oh, I will do that for you on the next push! Sorry about that

File: book_webframe/client/src/components/Bookpost.js [hunk 1]
```diff
@@ -86,6 +86,64 @@ export class Bookpost extends Component
...
+            this.setState({
+                bookCondition: e.target.value,
+                bookPrice: "20"
+            })    
+        };
+
+        if(e.target.value === 'Fair'){
+            this.setState({
+                bookCondition: e.target.value,
+            })
```
Comment (pamelayin) [inline book_webframe/client/src/components/Bookpost.js:114]: You're missing book price for this one

**Review by pamelayin — APPROVED** [review]
One thing I would recommend is to put the names same as db ,such as bookPrice, bookAvail to book_points, and available. it may be easier to track so you can do like book_points : this.state.bookPrice.

**Comment by pamelayin** [comment]
> > Not a show stopper, but maybe add 'rejected' variable in swap object?
> 
> Oh, I will do that for you on the next push! Sorry about that

I might have done that while i was fixing mine lol

File: book_webframe/client/src/components/Bookpost.js (hunk 1 above)
Comment (zldzksk1) [reply book_webframe/client/src/components/Bookpost.js:114]: Oh for the Fair, I set the default price, so it doesn't have to be updated onChange event

**Pull request merged**